### PR TITLE
Fix: Catalog: limit overview page #251

### LINKED ISSUE #1656: Reduce the number of cards shown on initial page load
### Description

when [meshery.io/catalog](https://meshery.io/catalog) loads, it displays all the patternCards at once. 
### Expected Behavior

It should show only the first 10 cards.
Visitors can then use `Prev` and `Next` buttons to display cards.
### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by abhishekmmgn**
**Description**

This PR fixes #1656 
fixes layer5labs/meshery-extensions-packages/issues/251

Limit the catalog cards count.

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by vishalvivekm — APPROVED** [review]

**Review by leecalcote — DISMISSED** [review]
Good. This gets us an immediate fix. This area is in need of further cleanup.

**Comment by leecalcote** [comment]
Actually, in the deployment preview the snapshot images still don't load on any of the 12 cards that are displayed....

**Comment by leecalcote** [comment]
Let's try 9 and see how it performs...

**Comment by abhishekmmgn** [comment]
> Actually, in the deployment preview the snapshot images still don't load on any of the 12 cards that are displayed....

Do we have any preview?

**Comment by vishalvivekm** [comment]
> > Actually, in the deployment preview the snapshot images still don't load on any of the 12 cards that are displayed....
> 
> Do we have any previews?
> 

Here @abhishekmmgn 
https://github.com/meshery/meshery.io/pull/1670#issuecomment-2022012422

**Comment by abhishekmmgn** [comment]
Am I getting this wrong?

We are talking about these images, aren't we?

image

File: _includes/patterns-filter.html [hunk 1]
```diff
@@ -361,9 +361,9 @@
 
     // If no checkbox is selected, show all pattern cards and enable the "All" radio button
     if (!filterArray.length) {
-      for (let j = 0; j < patternCards.length; j++) {
-        patternCards[j].style.display = "block";
-      }
+      // for (let j = 0; j < patternCards.length; j++) {
```
Comment (sudhanshutech) [inline _includes/patterns-filter.html:364]: why commented this?

**Comment by sudhanshutech** [comment]
@abhishekmmgn In the cards, we aim to display actual snapshots or screenshots of the designs. If there isn't any snapshot available, a default tree-like structure image is shown. However, there is a bug preventing the snapshots from displaying in dark mode. Nevertheless, you will be able to see them in light mode.
image

File: _includes/patterns-filter.html (hunk 1 above)
Comment (abhishekmmgn) [reply _includes/patterns-filter.html:364]: The loop was displaying all the patternCards on the 1st load.

**Pull request merged**